### PR TITLE
chore(backend): update api package name

### PR DESCRIPTION
**Pull request by Linchin**
**Description of your changes:**
Updated the v2 api package name to `kubeflow.pipelines.backend.api.v2beta1`.

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Review by chensun — COMMENTED** [review]
/lgtm

The change has no impact on generated python client?

**Comment by Linchin** [comment]
> /lgtm
> 
> The change has no impact on generated python client?

Indeed, there was no change.

File: backend/api/v2beta1/experiment.proto [hunk 1]
```diff
@@ -15,7 +15,7 @@
 syntax = "proto3";
 
 option go_package = "github.com/kubeflow/pipelines/backend/api/v2beta1/go_client";
-package v2beta1;
+package kubeflow.pipelines.backend.api.v2beta1;
```
Comment (gkcalat) [inline backend/api/v2beta1/experiment.proto:18]: Hi @Linchin. This should be already in the [master branch](https://github.com/kubeflow/pipelines/blob/master/backend/api/v2beta1/experiment.proto#L18) after [this change](https://github.com/kubeflow/pipelines/pull/8533/files#diff-ca1b12d8b07102930c2c7d54e2d934b84f2075b00500ba25fea2f7a67e4fe927R18).

File: backend/api/v2beta1/error.proto [hunk 2]
```diff
@@ -15,7 +15,7 @@
 syntax = "proto3";
 
 option go_package = "github.com/kubeflow/pipelines/backend/api/v2beta1/go_client";
-package v2beta1;
+package kubeflow.pipelines.backend.api.v2beta1;
```
Comment (gkcalat) [inline backend/api/v2beta1/error.proto:18]: As we decided to use `google.rpc.status.proto`, `error.proto` can actually be removed.

**Review by gkcalat — COMMENTED** [review]
Hi @Linchin. please, rebase on the `master` branch. I think this already exists in the `master`, but please correct me if I am wrong.

**Pull request closed without merging**